pipelines/README.md

```diff
@@ -0,0 +1,105 @@
+# Integration Test Pipelines
+
+Follow the instructions [here](https://github.com/RedHatInsights/bonfire-tekton?tab=readme-ov-file#add-the-integration-test-scenario-to-your-application) for getting setup with one of the following integration test pipelines and then configure it to use one of the pipelines on this page.  
+
+- [Basic](#basic)
+- [Basic with no IQE](#basic-with-no-iqe)
+- [Frontend Testing Pipeline](#frontend-testing-pipeline) 
+
+## Basic
+
+### Description
+
+TODO: Write description
+
+
+## Basic with no IQE
+
+### Description
+
+TODO: Write description
+
+
+## Frontend Testing Pipeline
+
+### Description
+
+The frontend testing pipeline (fe_testing.yaml) allows app teams to create a custom task at the `.tekton` directory of an application's repository. [Here](https://github.com/RedHatInsights/insights-chrome/tree/master/.tekton/run-tests-tasks.yml) is an example of this custom task.
+
+### Instructions
+
+1. Follow the instructions [here](https://github.com/RedHatInsights/bonfire-tekton/blob/main/README.md#add-the-integration-test-scenario-to-your-application) for setting up an integration test pipeline but use the following configuration for the FE testing pipeline: 
+
+```
+---
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  labels:
+    test.appstudio.openshift.io/optional: "true"
+  name: <app name>-bonfire-tekton
+  namespace: <workspace name> 
+spec:
+  application: <app name> 
+  resolverRef:
+    params:
+      - name: url
+        value: https://github.com/RedHatInsights/bonfire-tekton.git
+      - name: revision
+        value: main
+      - name: pathInRepo
+        value: pipelines/fe_testing.yaml
+    resolver: git
+  params:
+    - name: PROJECT_URL
+      value: <github project url> 
+    - name: APP_NAME
+      value: <app name> 
+    - name: COMPONENTS
+      value: <app name> 
+    - name: BONFIRE_COMPONENT_NAME
+      value: <app name> 
+    - name: COMPONENT_NAME
+      value: ""
+    - name: DEPLOY_FRONTENDS
+      value: "true"
+```
+
+2. Create the test task in your Github project within the `.tekton` directory:
```

Comment:
What should the name of the file be? I'm also a bit confused because I currently don't see a file with a 'Task' in it within the insights-chrome repo.  Linking directly to an example will help.

Reply:
Oh good point, let me add the name of the file that it has to be. Also waiting on them to merge this in: https://github.com/RedHatInsights/insights-chrome/pull/3041

There should be an example of this posted, I will add it here.